index.html

```diff
@@ -4,28 +4,171 @@
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-sm-12 mt-5">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th>Planet</th>
+                <th>Distance</th>
+                <th>Survivability</th>
+                <th>How Long To Travel</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Mars</td>
+                <td>54.6 million km</td>
+                <td>Maybe</td>
+                <td>300 Days</td>
+              </tr>
+              <tr>
+                <td>Jupiter</td>
+                <td>588 million km</td>
+                <td>No</td>
+                <td>Depends on Orbit (3-6 Years)</td>
+              </tr>
+              <tr>
+                <td>Saturn</td>
+                <td>1.2 billion km</td>
+                <td>No</td>
+                <td>Depends on Orbit (3-6 Years)</td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+      <div class="row px-4 py-5 mt-5 ruled-section">
+        <div class="col-sm-6 mt-3">
+          <h2>Let's Go For A Walk</h2>
+          <p>The Earth was small, light blue, an so touchingly alone, our home that must be defended like a holy relic. The Earth was absolutely round.</p>
+          <p>It suddenly struck me that that tiny pea, pretty and blue, was the Earth. I put up my thumb and shut one eye, and my thumb blotted out the planet Earth</p>
+        </div>
+        <div class="col-sm-6 mt-3">
+          <img src="../img/space-walk.jpg" class="img-fluid rounded" alt="Space Walk">
+        </div>
+      </div>
+      <div class="row mt-5">
+        <div class="col-sm-8">
+          <h3>Ready To Go?</h2>
+          <form action="">
+            <div class="form-group">
+              <label for="Name">Name</label>
+              <input type="email" class="form-control" name="" id="inputName" placeholder="Enter name">
+            </div>
+            <div class="form-group">
+              <label for="Name">Phone</label>
+              <input type="email" class="form-control" name="" id="inputPhone" placeholder="Enter phone">
+            </div>
+            <div class="form-group">
+              <label for="Name">Email</label>
+              <input type="email" class="form-control" name="" id="inputEmail" placeholder="Enter email">
+            </div>
+            <div class="form-check mt-1">
+              <input type="checkbox" name="promotional" id="promotionalCheck" class="form-check-input">
+              <label for="promotionalCheck" class="form-check-label">I would like to receive promotional emails</label>
+            </div>
+            <div class="form-check mt-1">
+              <input type="checkbox" name="noEmail" id="noEmailCheck" class="form-check-input">
```

Comment:
A minute detail that can be fixed with DOM, this may not be the best choice for checkbox only because you're able to click on both!